src/libs/actions/User.js

```diff
@@ -83,17 +80,28 @@ function updatePassword(oldPassword, password) {
  * @param {String} message optional reason for closing account
  */
 function closeAccount(message) {
-    DeprecatedAPI.User_Delete({message}).then((response) => {
-        console.debug('User_Delete: ', JSON.stringify(response));
-
-        if (response.jsonCode === 200) {
-            Growl.show(Localize.translateLocal('closeAccountPage.closeAccountSuccess'), CONST.GROWL.SUCCESS);
```

Comment:
We can delete this string translation now, right?

Reply:
Niceeee i think I remember thinking about doing this 👀 😅